index.js

```diff
@@ -26,12 +39,33 @@
  @Test - Make a function call of  `showMeTheMoney()`
 */
 
+function showMeTheMoney(x) {
+    
+    for(let i = 0; i <= x; i++){ // here I am looping from 1 until 200
+        // setting up condition for the if statement
+        if (i > 190 && i <= x) { // condition is: if the number fed is more than 190, console log it.
+            console.log(i);
+        }
+    };  
+};
+showMeTheMoney(200)
+
+
 /* 
  @Challenge 04 - Write a function named `showTheFifties` that loops from 1 to 200. 
  Use a condition in the loop to console.log the number when it is over 49 and under 60
  @Example - Expect to see 50, 51, 52, 53, 54, 55, 56, 57, 58, 59 in the terminal
  @Test - Make a function call of  `showTheFifties()`
 */
+function showTheFifties(x) {
+    let i;
```

Comment:
Extra `let` statement here on 61. Not harmful. Not needed though so making note.